starwars/src/components/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import CardBody from "./CardBody";
+
+const Card = props => {
+  return (
+    <div className="container">
+      {props.characters.map((x, i) => {
+        return (
+          <div className="card" key={i}>
+            <h1 className="card__title">{props.characters[i].name}</h1>
```

Comment:
I would have this line and the one below in your CardBody. This way when styling and when other developers look at the code it will all be in one place.